src/main/content/_layouts/guide-multipane.html

```diff
@@ -119,7 +119,7 @@ <h1 id="guide-title">{{ page.title }}</h1>
                     <div id="github-clone-popup" tabindex="0">
                         <div id="github-clone-popup-title">Git clone this repo to get going right away:</div>
                         <div id="github-clone-popup-repo">
-                                git clone https://github.com/Kabanero-io/guide-{{page.url | replace: '/guides/', ''}}.git
+                                git clone https://github.com/Kabanero-io/guide-{{page.url | replace: '/guides/', '' | remove: '/'}}.git
```

Comment:
is there a page.url that ends in 2 slashes? `//` for example? I would try to fix the problem there first rather than down the line here

Reply:
page.url is actually correct it only ends with one / so page.url would print something like `/guides/app-monitoring/` and we're only tryig to parse out the guide name so `app-monitoring` we removed the `/guides/` section but forgot to remove the remaning / at the end before adding `.git`

Reply:
ah got it, I understand now, thanks!